MobileCenterAnalytics/MobileCenterAnalyticsTests/MSSessionTrackerTests.m

```diff
@@ -196,4 +196,38 @@ - (void)testStartSessionOnAppForegrounded {
                             withPriority:MSPriorityDefault]);
 }
 
+- (void)testOnProcessingLogWithPriority {
+
+  // When
+  MSLogWithProperties *log = [MSLogWithProperties new];
+
+  // Then
+  XCTAssertNil(log.sid);
+  XCTAssertNil(log.toffset);
+
+  [self.sut onProcessingLog:log withPriority:MSPriorityDefault];
+
+  // And then
+  XCTAssertNil(log.toffset);
+  XCTAssertEqual(log.sid, self.sut.sessionId);
+
+  // When
+  log.toffset = 0;
+
```

Comment:
No, we can't, because we should check what we get first session id if we have the smallest time offset.